sormas-ui/src/main/java/de/symeda/sormas/ui/configuration/DevModeView.java

```diff
@@ -1419,4 +1689,182 @@ public void setPercentageOfCases(int percentageOfCases) {
 		}
 
 	}
+
+	private static class SampleGenerationConfig {
+
+		private static int sampleCount;
+		private static SamplePurpose samplePurpose;
+		private static LocalDate startDate;
+		private static LocalDate endDate;
+		private static SampleMaterial sampleMaterial;
+		private static String sampleMaterialText;
+		private static FacilityReferenceDto laboratory;
+
+		private static boolean externalLabOrInternalInHouseTesting = false;
+		private static boolean requestPathogenTestsToBePerformed = false;
+		private static boolean requestAdditionalTestsToBePerformed = false;
+		private static boolean sendDispatch = false;
+		private static boolean received = false;
+		private static String comment;
+
+		private static Disease disease;
+		private static RegionReferenceDto region;
+		private static DistrictReferenceDto district;
+
+		SampleGenerationConfig() {
+			loadDefaultConfig();
+		}
+
+		public static void loadDefaultConfig() {
```

Comment:
this static methods should create a new instance of SampleGenerationConfig instead of making SampleGenerationConfig static